migrations/001_hello_joe/up.sql

```diff
@@ -1,13 +1,19 @@
-create table packages (
-  name varchar(100) primary key,
-  imported_at timestamp not null default current_timestamp
+create table packages
+(
+    name        varchar(100) primary key,
+    updated_at  timestamp      not null default current_timestamp,
+    imported_at timestamp      not null default current_timestamp,
+    links       jsonb          not null default '{}'::jsonb,
+    licenses    varchar(255)[] not null default array[]::varchar(255)[],
+    description text           not null default 'My Description'
 );
 
-create table previous_hex_api_scan (
-  id boolean primary key default true,
-  scanned_at timestamp not null,
+create table previous_hex_api_scan
+(
+    id         boolean primary key default true,
+    scanned_at timestamp not null,
 
-  -- We use a constraint to enforce that the id is always the value `true` so
-  -- now this table can only hold one row.
-  constraint previous_hex_api_scan_singleton check (id)
+    -- We use a constraint to enforce that the id is always the value `true` so
+    -- now this table can only hold one row.
+    constraint previous_hex_api_scan_singleton check (id)
 )
```

Comment:
Has this file been formatted with an auto formatter? I have been looking for one for SQL

Reply:
Yeah I ran it though datagrip, but just reformatted with the suggest from @inoas from discord